src/components/CandleCharts.js

```diff
@@ -0,0 +1,83 @@
+import { VictoryChart,VictoryCandlestick,VictoryAxis,VictoryTheme,VictoryZoomContainer,VictoryScatter,VictoryCursorContainer } from "victory";
+import { useEffect,useState } from "react";
+const CandleCharts=()=>{
+
+
+  const axios = require('axios')
+  const baseUrl = "https://api-pub.bitfinex.com/v2/";
```

Comment:
Please create separate file for constants

Reply:
Comment addressed.Please review